src/chrome/breakOnLoadHelper.ts

```diff
@@ -67,12 +66,16 @@ export class BreakOnLoadHelper {
         }
     }
 
+    private getPausedScriptUrlFromId(scriptId: string): string {

```

Comment:
Why just a "paused" script?

Reply:
You are right. Renaming to getScriptUrlFromId